R/families.R

```diff
@@ -113,26 +113,72 @@
 #'   zero-inflated and hurdle models. These models can be very helpful when there
 #'   are many zeros in the data (or ones in case of one-inflated models)
 #'   that cannot be explained by the primary distribution of the response.}
-#'   }
 #'
-#'   Below, we list all possible links for each family.
-#'   The first link mentioned for each family is the default.
-#'   \itemize{
-#'   \item{Families \code{gaussian}, \code{student}, \code{skew_normal},
-#'   \code{exgaussian}, \code{asym_laplace}, and \code{gen_extreme_value}
-#'   support the links (as names) \code{identity}, \code{log}, \code{inverse},
-#'   and \code{softplus}.}
+#'
+#'   \item{Family \code{xbetax} allows to estimate extended-support
```

Comment:
Now a bit long perhaps (sorry). I will adjust this a bit before merging.

Reply:
😄 OK. Happy to take a look at the edited version before merging